abstract/abstract.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Processing potential fields data with Fatiando a Terra'
```

Comment:
we need to update the title here "Comparing strategies for assessing uncertainty with geophysical inversions for mineral exploration"